Client/YooChooseNotifier.php

```diff
@@ -161,29 +162,28 @@ public function updateContent($contentId)
      */
     public function deleteContent($contentId)
     {
+        $content = $this->contentService->loadContent($contentId);
+
         try {
-            if (!in_array($this->getContentTypeIdentifier($contentId), $this->options['included-content-types'])) {
-                // this Content is not intended to be submitted because ContentType was excluded
+            if ($this->isContentTypeExcluded($content)) {
                 return;
             }
         } catch (NotFoundException $e) {
             // this is most likely a internal draft, or otherwise invalid, ignoring
             return;
         }
 
-        if (isset($this->logger)) {
-            $this->logger->info("Notifying YooChoose: delete($contentId)");
+        $this->log(sprintf('Notifying YooChoose: deleteContent(%s)', $content->id));
```

Comment:
`Recommendation Service`

Reply:
What happens when you delete a `Folder` with `children`? You get several notifications for the folder and the children?